recipes/beaupy/meta.yaml

```diff
@@ -28,10 +28,6 @@ requirements:
 test:
   imports:
     - beaupy
-  commands:
-    - pip check
```

Comment:
Can you comment on what is broken? Sometimes we have to fix dependencies downstream.